src/utils/db.ts

```diff
@@ -0,0 +1,48 @@
+import { Level } from "level";
+
+export type DBLevel = Level<string, string>;
+
+export default class DBService {
```

Comment:
nit: This name is a bit generic (but it depeds on Level db).

It would be great if you can use the Repository or DAO pattern to abstract the persistance. This way, this implementation will use Level DB, but others could use anything else.

This could have a repository or DAO layer, and then you create a `ContextRepository` interface, with a `ContextRepositoryLevel` or `ContextRepositoryImpl` (since we don't have more)